ts/features/itwallet/presentation/components/ItwPresentationDetailFooter.tsx

```diff
@@ -2,44 +2,108 @@ import {
   Chip,
   IOStyles,
   ListItemAction,
+  useIOToast,
   VSpacer
 } from "@pagopa/io-app-design-system";
 import React from "react";
-import { Linking, View } from "react-native";
+import { Alert, Linking, View } from "react-native";
 import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { useIODispatch } from "../../../../store/hooks";
 import { format } from "../../../../utils/dates";
-import { IssuerConfiguration } from "../../common/utils/itwTypesUtils";
+import { itwCredentialNameByCredentialType } from "../../common/utils/itwCredentialUtils";
+import { CredentialType } from "../../common/utils/itwMocksUtils";
+import { StoredCredential } from "../../common/utils/itwTypesUtils";
+import { itwCredentialsRemove } from "../../credentials/store/actions";
+
+// TODO: use the real credential update time
+const lastUpdateTime = new Date();
 
 type Props = {
-  lastUpdateTime: Date;
-  issuerConf: IssuerConfiguration;
+  credential: StoredCredential;
 };
 
-export const ItwPresentationDetailFooter = ({
-  lastUpdateTime,
-  issuerConf
-}: Props) => (
-  <View>
-    <ListItemAction
-      variant="primary"
-      icon="message"
-      label={I18n.t(
-        "features.itWallet.presentation.credentialDetails.actions.requestAssistance",
-        { authSource: issuerConf.federation_entity.organization_name }
-      )}
-      accessibilityLabel={I18n.t(
-        "features.itWallet.presentation.credentialDetails.actions.requestAssistance",
-        { authSource: issuerConf.federation_entity.organization_name }
-      )}
-      onPress={() =>
-        Linking.openURL(`mailto:${issuerConf.federation_entity.contacts?.[0]}`)
-      }
-    />
-    <VSpacer size={24} />
-    <Chip color="grey-650" style={IOStyles.selfCenter}>
-      {I18n.t("features.itWallet.presentation.credentialDetails.lastUpdated", {
-        lastUpdateTime: format(lastUpdateTime, "DD MMMM YYYY, HH:mm")
-      })}
-    </Chip>
-  </View>
-);
+export const ItwPresentationDetailFooter = ({ credential }: Props) => {
+  const dispatch = useIODispatch();
+  const navigation = useIONavigation();
+  const toast = useIOToast();
+
+  const handleRemoveCredential = () => {
+    dispatch(itwCredentialsRemove(credential));
```

Comment:
I think we can remove the cryptographic keys as well, with `deleteKey` from io-react-native-crypto. We save the key tag in every credential.

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6126/commits/072e4f5231368dc7554ae05fd529b0b7e9beb494